### ISSUE TITLE
TROCR's input is 384*384, is there are better method?

### ISSUE THREAD

**Issue by archwolf118** [microsoft/unilm#674]
Thank you for the wonderful work in TROCR. There are a lot of methods to try to combine language model, like transformer model, in OCR recognition model, and no one achieve better accuracy then TROCR.
But when I check  the code and papper, I found TROCR resize the image input to 384 * 384. As we know "ocr recognition task" always has 32 * (16~2000) image input. the width of text images is changed in large scope.For example If we resize the image from 32*2000 to  384*384, the text information will be distorted.
I think the biggest problem in attention method  (transformer model) use in OCR recognition task is the image size is changed in large scope. That is why CRNN (with Rnn model) can dominate OCR Industry from 2016.
So I want to know fixed 384*384 input is reasonable? And  is there are better method to trained the TROCR model with not fixed width text image?
Thank you!

**Comment by wenyinlong** [comment]
This Beit model is only used to extract image embedding, you can change the encoder "encoder_ hidden_ states" with other encoder model。

**Comment by wolfshow** [comment]
@archwolf118, we use 224x224 or 384x384 resolution for the input image because it is easier to load pre-trained models. It is definitely possible to use other resolutions in the pre-training/fine-tuning.

**Comment by CrasCris** [comment]
> @archwolf118, we use 224x224 or 384x384 resolution for the input image because it is easier to load pre-trained models. It is definitely possible to use other resolutions in the pre-training/fine-tuning.

But how can you set the resolutions, to use a different one insted of the the 384*384 ?

**Comment by channudam2002** [comment]
> > @archwolf118, we use 224x224 or 384x384 resolution for the input image because it is easier to load pre-trained models. It is definitely possible to use other resolutions in the pre-training/fine-tuning.
> 
> But how can you set the resolutions, to use a different one insted of the the 384*384 ?

I also want to know how to change the resolution too for fine-tunning. Thanks!
